pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -27,13 +27,21 @@ import (
 	"github.com/tektoncd/operator/pkg/reconciler/common"
 	"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektoninstallerset/client"
 	occommon "github.com/tektoncd/operator/pkg/reconciler/openshift/common"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
 	"knative.dev/pkg/logging"
 )
 
 const (
 	// manifests console plugin yaml directory location
 	routeRBACYamlDirectory  = "static/tekton-results/route-rbac"
 	internalDBYamlDirectory = "static/tekton-results/internal-db"
+	deploymentAPI           = "tekton-results-api"
+	apiContainerName        = "api"
+	boundSAVolume           = "bound-sa-token"
+	boundSAPath             = "/var/run/secrets/openshift/serviceaccount"
```

Comment:
Is this the default path which exist on all the openshift clusters?
Any doc reference to show about this path usage ?

Reply:
`The path to the service account token inside the pod. By convention, this is /var/run/secrets/openshift/serviceaccount/token for OpenShift Container Platform components.`

https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/cco-mode-sts.html

Same in Google GCS.

Reply:
We support only AWS or Google. So both have the same convention for `OpenShift Container Platform components`. 
I tested this in a CU cluster.

We will need to pass properties and modify when we add Azure but we don't support Azure anyway.